### PR TITLE
[mixin] Add cortex-gw-internal to dashboards

### PR DESCRIPTION
**Pull request by gouthamve**
We are now accepting data through the internal gateway as well. This updates the dashboards to reflect that.

No alerts or recording rules reference `cortex-gw`. 

Does this need a CHANGELOG entry?

### CODE REVIEW

**Review by dannykopping — COMMENTED** [review]
I'm a little bit unsure why we have `cortex-gw` in our mixins at all since this is proprietary Grafana Labs middleware; I'm not sure we should be adding the other variant here too.

I'll raise this with the team tomorrow

File: production/loki-mixin/dashboards/dashboard-utils.libsonnet [hunk 1]
```diff
@@ -167,7 +167,7 @@ local utils = import 'mixin-utils/utils.libsonnet';
       tooltip: { sort: 2 },  // Sort descending.
     },
   containerCPUUsagePanel(title, containerName)::
-    self.CPUUsagePanel(title, 'container="%s"' % containerName),
+    self.CPUUsagePanel(title, 'container=~"%s"' % containerName),
```
Comment (dannykopping) [inline production/loki-mixin/dashboards/dashboard-utils.libsonnet:170]: Curious why these changes were made? They seem unrelated to the purpose of the PR.
The compiled versions generally seem to not need the regex, and I'm concerned about containers like `query-scheduler` where `-` has a semantic meaning in regex.

**Review by dannykopping — APPROVED** [review]
Left a nit, approving to unblock
We will remove all mention of cortex-gw from our mixin, and move this into our own tooling

File: production/loki-mixin/dashboards/dashboard-utils.libsonnet (hunk 1 above)
Comment (gouthamve) [reply production/loki-mixin/dashboards/dashboard-utils.libsonnet:170]: You're right! Fixing it.

**Comment by dannykopping** [comment]
@gouthamve can you please recompile the mixins to make CI happy?
I'll merge after that

**Comment by gouthamve** [comment]
@dannykopping Its now done! Thanks!
